code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -63,8 +64,10 @@
 			apply_cuffs(C, user, remove_src)
 			to_chat(user, "<span class='notice'>You handcuff [C].</span>")
 			SSblackbox.record_feedback("tally", "handcuffs", 1, type)
-
-			add_attack_logs(user, C, "Handcuffed ([src])")
+			if(secure)
```

Comment:
Is it not better to check if the "breakouttime" is 0?

Reply:
oh. Yeah, keep it simple stupid moment.